x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/eql/build_alert_group_from_sequence.test.ts

```diff
@@ -43,19 +43,19 @@ describe('buildAlert', () => {
         sampleDocNoSortId('619389b2-b077-400e-b40b-abde20d675d3'),
       ],
     };
-    const alertGroup = buildAlertGroupFromSequence(
-      ruleExecutionLoggerMock,
-      eqlSequence,
+    const { shellAlert, buildingBlocks } = buildAlertGroupFromSequence({
+      ruleExecutionLogger: ruleExecutionLoggerMock,
+      sequence: eqlSequence,
       completeRule,
-      'allFields',
-      SPACE_ID,
-      jest.fn(),
-      completeRule.ruleParams.index as string[],
-      undefined,
-      PUBLIC_BASE_URL
-    );
-    expect(alertGroup.length).toEqual(3);
-    expect(alertGroup[0]).toEqual(
+      mergeStrategy: 'allFields',
+      spaceId: SPACE_ID,
+      buildReasonMessage: jest.fn(),
+      indicesToQuery: completeRule.ruleParams.index as string[],
+      alertTimestampOverride: undefined,
+      publicBaseUrl: PUBLIC_BASE_URL,
+    });
+    expect(buildingBlocks.length).toEqual(2);
+    expect(shellAlert).toEqual(
```

Comment:
the shell alert used to be the last object in the array, not the first - so alertGroup[0] is buildingBlocks[0], etc